keras/src/ops/numpy.py

```diff
@@ -2669,6 +2669,32 @@ def exp(x):
     return backend.numpy.exp(x)
 
 
+class Exp2(Operation):
+    def call(self, x):
+        return backend.numpy.exp2(x)
+
+    def compute_output_spec(self, x):
+        dtype = backend.standardize_dtype(x.dtype)
+        if "int" in dtype or dtype == "bool":
+            dtype = backend.floatx()
+        return KerasTensor(x.shape, dtype=dtype)
+
+
+@keras_export(["keras.ops.exp2", "keras.ops.numpy.exp2"])
+def exp2(x):
+    """Calculate the exponential of all elements in the input tensor.
```

Comment:
The docstring looks inaccurate, it should something like "Calculate `2**p` for all `p` in the input tensor `x`."